src/qt/test/paymentservertests.cpp

```diff
@@ -185,7 +185,8 @@ void PaymentServerTests::paymentServerTests()
     tempFile.open();
     tempFile.write((const char*)randData, sizeof(randData));
     tempFile.close();
-    QCOMPARE(PaymentServer::readPaymentRequestFromFile(tempFile.fileName(), r.paymentRequest), false);
```

Comment:
If you want to be able to test a private methods you could use a friend class / function. E.g. inside the class do
```
friend class PaymentServerTester;
```
Then do the testing in a class PaymentServerTester.